# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1315937203 14400
#      Tue Sep 13 14:06:43 2011 -0400
# Node ID e255f8c3c2ff7da4dbdfa8e78829cfd1b725b970
# Parent  edb0150647324e32e476aa696b7368974cb00c55
maint: use gnulib opendir and closedir modules

* booststrap.conf (gnulib_modules): Include opendir and closedir in
the list.

diff --git a/build-aux/bootstrap.conf b/build-aux/bootstrap.conf
--- a/build-aux/bootstrap.conf
+++ b/build-aux/bootstrap.conf
@@ -15,16 +15,17 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 
 # gnulib modules used by this package.
 gnulib_modules="
   c-strcase
   copysign
+  closedir
   crypto/md5
   fclose
   fcntl
   filemode
   fnmatch
   fopen
   fflush
   fseek
@@ -37,16 +38,17 @@ gnulib_modules="
   link
   lstat
   mkdir
   mkfifo
   mkstemp
   mktime
   nanosleep
   nproc
+  opendir
   pathmax
   progname
   readlink
   rename
   rmdir
   round
   roundf
   sigaction
